code/ui/Nameplate.cs

```diff
@@ -28,16 +29,18 @@ public override void Tick()
                 .Ignore(player)
                 .Run();
 
-            bool targetFound = trace.Hit && trace.Entity is TTTPlayer;
+            bool validHit = false;
```

Comment:
Could it just be

```csharp
bool validHit = trace.Hit && trace.Entity is TTTPlayer target
if (validHit) {
...
}
SetClass("hide", !validHit);
```

Reply:
Doesn't work sadly, already tried, but if connected `trace.Entity is TTTPlayer` via `&&`, it seems that `target` isn't available anymore.

Reply:
huh odd.

Reply:
I threw it into Trello as well so we keep track on it.